randovania/data/json_data/prime3.txt

```diff
@@ -1530,13 +1789,20 @@ Asset id: 1872319291341539278
 > Dock to Jungle Generator; Heals? False
   * Normal Door to Jungle Generator/Dock to Generator Hall South
   > Dock to South Jungle Court
+      After Generator Hall South Bridge
+  > Event - Destroy Bridge
       Trivial
 
 > Dock to South Jungle Court; Heals? False
   * Normal Door to South Jungle Court/Dock to Generator Hall South
   > Dock to Jungle Generator
       Trivial
 
+> Event - Destroy Bridge; Heals? False
+  * Event Generator Hall South Bridge
+  > Dock to Jungle Generator
+      Trivial
+
```

Comment:
I'm thinking this event isn't necessary, as it's just shooting a charge shot it or missile. So you can remove this event and node, and fix the connections, trivial for now